.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: build
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+
+    env:
+      # Configures freight
+      POSTGRES_ENV_POSTGRES_USER: postgres
+      POSTGRES_ENV_POSTGRES_PASSWORD: postgres_password
+      POSTGRES_ENV_POSTGRES_DB: test_freight
+      POSTGRES_PORT_5432_TCP_ADDR: 127.0.0.1
+
+      # Configures createdb command
+      PGHOST: 127.0.0.1
+      PGUSER: postgres
+      PGPASSWORD: postgres_password
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_DB: ${{ env.POSTGRES_ENV_POSTGRES_DB }}
+          POSTGRES_USER: ${{ env.POSTGRES_ENV_POSTGRES_USER }}
+          POSTGRES_PASSWORD: ${{ env.POSTGRES_ENV_POSTGRES_PASSWORD }}
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: volta-cli/action@v1
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.8'
+
+      - name: Install dependencies
```

Comment:
You should cache. Also, wheel should be installed (GHA setup-python doesn't have it):

```
Using legacy 'setup.py install' for alembic, since package 'wheel' is not installed.
Using legacy 'setup.py install' for rq, since package 'wheel' is not installed.
Using legacy 'setup.py install' for loremipsum, since package 'wheel' is not installed.
Using legacy 'setup.py install' for blinker, since package 'wheel' is not installed.
```

Reply:
Sure. I'll cache and install wheel